app/services/classroom/page.tsx

```diff
@@ -0,0 +1,169 @@
+    image: "/images/featured-carousel/provident.png",
+    profile: "Brown School of Public Health"
+  },
+  {
+    title: "USING MATLAB ON OSCAR",
+    category: "Statistics",
+    description: "Description of Matlab needed.",
+    image: "/images/featured-carousel/provident.png",
+    profile: "Brown School of Public Health"
+  },
+  {
+    title: "INTO TO STATISTICAL ANALYSIS WITH R ON OSCAR",
+    category: "Statistics",
+    description: "Description of R needed.",
+    image: "/images/featured-carousel/provident.png",
+    profile: "High Performance Computing"
+  },
+]
+
+const profiles = [
+  {
+  icon: <UserIcon className="w-6 h-6" />,
+  name: "Brown School of Public Health",
+  organization: "People, Place and Health Collective",
+  websiteUrl: "https://provident-study.org",
+  viewMoreUrl: "/provident",
+  },
+  {
+    icon: <UserIcon className="w-6 h-6" />,
+    name: "High Performance Computing",
+    organization: "Center for Computation and Visualization",
+    websiteUrl: "https://ccv.brown.edu",
+    viewMoreUrl: "/about/us",
+  },
+]
+
+export default async function ClassroomSupport() {
+    return (
+      <div className="w-full">
+        
+        <div className="relative w-full flex flex-col">
+          <div className="bg-purple-900">
+            <Hero image={"/images/hero-subroutes.jpeg"}>
+              <div className="relative flex-1 flex items-start w-full px-6 md:px-24 bg-gradient-to-t from-black/0 via-black/10 to-black/65 z-5">
+                <div className="absolute top-[12%] flex flex-col text-white space-y-6">
+                  <TextAnimate className="font-bold text-6xl md:text-8xl">
+                    Classroom Support
+                  </TextAnimate>
+                  <p className="text-4xl font-semibold">
+                    CCV services to help faculty in the classroom. We can provide tutorial or give you access to cutting edge technology for teaching with code
+                  </p>
+                </div>
+              </div>
+            </Hero>
+          </div>
+        </div>
+
+      {/* In Class Tutorials */}
+      <section className="content-wrapper pt-24 px-14 lg:px-36">
+        <SectionHeader title="In-Class Tutorials" align="center" />
+        <Card className="w-full border-none shadow-none rounded-none">
+          <CardContent className="mx-auto flex items-center px-6">
+            <p className="text-black text-xl pb-8">
+              CCV offers a variety of tutorials to provide students with experience using Brown's HPC systems. CCV staff members provide students with an overview of the topic and guide them through a series of hands-on activities. Tutorials can range from the basics of using HPC systems to the use of specific applications on Brown's HPC systems.
+            </p>
+          </CardContent>
+        </Card>
+        <FeaturedCarousel carouselData={featuredCarouselData} profiles={profiles} />
+      </section>
+
+      {/* Student Accounts */}
```

Comment:
Can we also add in two buttons at the bottom of this section?

Learn More: `https://docs.ccv.brown.edu/oscar/account-types`
Request Student Accounts:  `mailto:support@ccv.brown.edu`